Flepper.QueryBuilder/Utils/CacheUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Linq;
+
+namespace Flepper.QueryBuilder.Utils
+{
+    internal class CacheUtils
```

Comment:
The class name is redundant since it is already within the namespace utils it does not need to have a suffix called utils, I would try to put something with _Cache_ or _CacheObjects_